### ISSUE TITLE
Can multivsos do weighted SOS decomposition?

### ISSUE THREAD

**Issue by OliveS9** [magronv/multivsos#1]
Could you please let me know if multivsos can do weighted SOS decompositions, and if not have you developed any usable software package that can do weighted SOS decomposition? Many thanks.

**Comment by magronv** [comment]
Yes, one can compute weighted SOS decompositions with multivsos. 
Please go to https://gricad-gitlab.univ-grenoble-alpes.fr/magronv/RealCertify for the current maintained version.

Best,

Victor
